packages/gen-typescript-declarations/src/test/goldens/polymer-3/lib/elements/array-selector.d.ts

```diff
@@ -191,6 +186,9 @@ declare class ArraySelector extends
   Polymer.Element) {
 }
 
-interface HTMLElementTagNameMap {
-  "array-selector": ArraySelector;
+declare global {
```

Comment:
Does this work if the containing file does not contain any imports or exports?

Reply:
I think you can safely add an `export {}` line to a file to force it to be a module without otherwise changing semantics

Reply:
Yeah, it does not work unless tsc thinks its a module. Added an `export {}` if there aren't already some exports.